com.obsproject.Studio.Plugin.ScaleToSound.metainfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.ScaleToSound</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>Scale To Sound Plugin</name>
+  <summary>Make a source scale reactively to the sound of an audio source</summary>
```

Comment:
Allow me to suggest the following: `Scale sources according to the sound of an audio source`